bootcamp/tutorials/integration/llama_agents_milvus.ipynb

```diff
@@ -0,0 +1,569 @@
+{
```

Comment:
<p> This repo contains the core components of Milvus Lite.</p><p>-- this is not related to this notebook.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1401/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>